source/delegate/test/Delegate.js

```diff
@@ -50,6 +53,9 @@ contract('Delegate Integration Tests', async accounts => {
   }
 
   before('Setup', async () => {
+    const snapShot = await takeSnapshot()
+    snapshotId = snapShot['result']
+
```

Comment:
We do use snapshots in our integration tests, but just 1 massve snapshot. You'll find snapshots in the integration tests for indexer, swap and wrapper. We don't snapshot every single test like we do in unit tests, however we take 1 snapshot at the beginning, and revert to that snapshot at the end of all the tests so that the slate is cleaned before the tests for the next contract are run.

Reply:
this is already done automatically by truffle.

Reply:
When why was it put into all the other integration tests?

Reply:
I thought I removed them, but i guess they were put back or we kept them? In any event if you run this without the time traveler you'll see everything passes. 

I guess it's technically doing the same thing as the truffle clean state. I just wanted to ensure that there wasn't any reverting of state in the integration tests that would give a false positive.

Reply:
Think it's a lack of trust of the clean room truffle guarantee. We've seen weird state behavior persisting.